srv/src/test/java/my/bookshop/AdminServiceTest.java

```diff
@@ -1,100 +1,57 @@
 package my.bookshop;
```

Comment:
Let's delete this test and CatalogServiceTest and just keep SimpleIntegrationTest.